common/app/views/fragments/email/support.scala.html

```diff
@@ -0,0 +1,69 @@
+@(page: model.Page)(implicit request: RequestHeader)
+
+<table class="subscribe sc" border="0" cellpadding="0" cellspacing="0" bgcolor="#ffffff">
```

Comment:
It would be really helpful if you could somehow share your knowledge on which style properties need to be inlined and for which vendors, and which properties it is safe to add to CSS. Is that something we could arrange (outside of this PR if it's easier)?